main.py

```diff
@@ -511,11 +529,12 @@ def configure(self) -> None:
         self.add_command(["close"], "close device `name`", self.command_device_close, AccessLevel.DEVICE, params="name")
         self.add_command(["commands", "c"], "list supported commands for device `name`", self.command_device_commands, AccessLevel.DEVICE, params="name")
         self.add_command(["dim", "d"], "dim device `name` by `number` percent", self.command_device_dim, AccessLevel.DEVICE, params="number name")
-        self.add_command(["events", "e"], "get recent events for device `name`", self.command_device_events, AccessLevel.SECURITY, params="name")
+        self.add_command(["events", "e"], "get last 20 events for device `name`", self.command_device_events, AccessLevel.SECURITY, params="name")
```

Comment:
Not sure where 20 is coming from.  The code is calling `self.hubitat.api.get_device_events(device.id)` and then iterating over them.  If Hubitat returns 20 events exactly and is never going to change this number than I guess the code is fine.  I like `recent` events more - less specific and still correct.

Reply:
ok